contracts/stellar-axelar-operators/src/migrate.rs

```diff
@@ -0,0 +1,35 @@
+use soroban_sdk::{panic_with_error, Address, Env, Vec};
+use stellar_axelar_std::interfaces::CustomMigratableInterface;
+
+use crate::error::ContractError;
+use crate::{storage, AxelarOperators};
+
+pub type MigrationData = <AxelarOperators as CustomMigratableInterface>::MigrationData;
+
+mod legacy_storage {
+    use soroban_sdk::Address;
+    use stellar_axelar_std::contractstorage;
+
+    #[contractstorage]
+    #[derive(Clone, Debug)]
+    enum LegacyDataKey {
+        #[instance]
+        #[status]
+        Operators { account: Address },
+    }
+}
+
+impl CustomMigratableInterface for AxelarOperators {
```

Comment:
Unrelated: these storage types need to be changed to use `contractstorage`
https://github.com/axelarnetwork/axelar-cgp-stellar/blob/main/packages/stellar-axelar-std/src/interfaces/mod.rs#L39